fdbserver/BlobGranuleValidation.actor.cpp

```diff
@@ -178,7 +178,7 @@ ACTOR Future<Void> clearAndAwaitMerge(Database cx, KeyRange range) {
 	state int reClearInterval = 1; // do quadratic backoff on clear rate, b/c large keys can keep it not write-cold
 	loop {
 		try {
-			Standalone<VectorRef<KeyRangeRef>> ranges = wait(tr.getBlobGranuleRanges(range));
+			Standalone<VectorRef<KeyRangeRef>> ranges = wait(tr.getBlobGranuleRanges(range, 1000));
```

Comment:
Technically since this is just checking whether there is one range or not, if this limit was the number of range limit instead of the row limit, we could just use 2 as the limit.
As rowLimit i think you're right in that we need to pass a much larger value though

Reply:
Done.